components/notificacoes.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Componentes de Notificação</title>
+    <link rel="stylesheet" href="css/components/notificacoes.css"/>
+</head>
+<body>
+    <!-- Templates de notificação (não visíveis inicialmente) -->
+    <div id="notificationSuccess" class="notification notification-success">
+        <div class="notification-icon">✓</div>
+        <div class="notification-content">
+            <h3 class="notification-title">Sucesso</h3>
+            <p class="notification-message">Produto cadastrado com sucesso!</p>
+        </div>
+    </div>
+
+    <div id="notificationError" class="notification notification-error">
+        <div class="notification-icon">✕</div>
+        <div class="notification-content">
+            <h3 class="notification-title">Erro ao cadastrar</h3>
+            <p class="notification-message">O campo <span id="errorField">Nome</span> é obrigatório.</p>
+        </div>
+    </div>
+
+    <script src="js/modules/notificacoes.js"></script>
```

Comment:
O que comentei anteriormente ocorre aqui também.

Se você tentar usar o caminho direto como js/modules/notificacoes.js sem o ../, o código tentará procurar o arquivo JS dentro da pasta components, o que resultará em um erro de caminho incorreto.

```suggestion
    <script src="../js/modules/notificacoes.js"></script>
```